tests/integration/utils.py

```diff
@@ -289,6 +289,56 @@ def test_endpoint(self, endpoint, resource, response_code, query_params=None,
     return response
 
 
+def test_filter_params(self, endpoint, filter_type, valid_tests,
+                       invalid_tests):
+    PARK_RES = 'ParkResource'
```

Comment:
Not a big deal, but we usually put constant variables right after module import sections. You can just define this as a normal variable here.